### DISCUSSION TITLE
Guide to picking the best model for your dataset?

### DISCUSSION THREAD

**Discussion by mazatov** [PaddlePaddle/PaddleOCR#7910] [Q&A]
I was wondering if there is a "common sense" guide for picking the best model for your dataset. I'm training a model on my own dataset that has 1 or 2-digit numbers with low resolutions (20-40 pixels). I started out with the default CVTR + MobileNetV1Enhace (`en_PP-OCRv3_rec.yml`) algorithm, which performs pretty while overfitting the training dataset. 

I was wondering if there is a guide on how to pick the best model for your problem. I'd like to start with a model that makes the most sense before trying to tune hyperparameters.

**Comment by bely66** [comment]
I think you might use the simplest model (I think CRNN), with their optimizations it should be pretty accurate for your case and use it as a baseline in Accuracy and Speed
Then you can use more complex models like SAR, ABINET or ppocrv3
If there's any value from using more complex models check your time requirements and decide accordingly
